### ISSUE TITLE
Accessibility - Color contrast does not meet enhanced requirement

### ISSUE THREAD

**Issue by id2win** [GSA/idmanagement.gov#516]
Issue level AAA

Why? 
Text and background colors that conflict or have too little contrast between each other can cause readability issues for users who are color-blind or have low vision (vision loss equivalent to approx. 20/80 vision).

There are 56 instances of color contrast enhancements. Recommended value is 7:1, currently below this value. Details for exact issues here - https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?PageId=38246420896&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r66/failed
